core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -64,12 +62,13 @@ var StoryEditorPage = function() {
     by.css('.protractor-test-publish-story-button'));
   var unpublishStoryButton = element(
     by.css('.protractor-test-unpublish-story-button'));
+  var dragAndDrop = async function(fromElement, toElement) {
+    await browser.executeScript(dragAndDropScript, fromElement, toElement);
+  };
```

Comment:
Can this be made into a utility in `general.js`? It seems generally useful

Reply:
Yeah it'd be a lot better. Done.